### ISSUE TITLE
Path of compose file in READ  "no such file or directory"

### ISSUE THREAD

**Issue by wenwenchenbosch** [GENIVI/iot-event-analytics#96]
in https://github.com/GENIVI/iot-event-analytics/blob/develop/src/sdk/javascript/examples/integrations/kuksa.val/README.md

```
docker-compose -f docker-compose.mosquitto.yml -f docker-compose.platform.yml -f ../src/sdk/javascript/examples/integrations/kuksa.val/docker-compose.kuksa.val2iotea.yml --env-file ../src/sdk/javascript/examples/integrations/kuksa.val/.env --project-name kuksa-val-stack up --remove-orphans --build
```

`src/sdk/javascript/examples/integrations/kuksa.val/docker-compose.kuksa.val2iotea.yml` does not exist

should be changed to 
```
docker-compose -f docker-compose.mosquitto.yml -f docker-compose.platform.yml -f ../src/sdk/javascript/examples/integrations/kuksa.val/docker-compose.kuksa2iotea.yml --env-file ../src/sdk/javascript/examples/integrations/kuksa.val/.env --project-name kuksa-val-stack up --remove-orphans --build
```

**Comment by wenwenchenbosch** [comment]
additionally I got the error:

```
ERROR: Version mismatch: file ./docker-compose.mosquitto.yml specifies version 3.5 but extension file ./docker-compose.platform.yml uses version 3.6
```

**Comment by llakie** [comment]
We  will update the documentation and also synchronize the specified yml versions for docker-compose. Thank you for your input.

**Comment by llakie** [comment]
Will be fixed in  PR #97

**Comment by llakie** [comment]
Merged
